### DISCUSSION TITLE
Make it useable in the EU for selfhosters

### DISCUSSION THREAD

**Discussion by devcomputer** [OpenSignLabs/OpenSign#811] [Ideas]
I gave OpenSign a test run and cant find a user management at all; no admin, nothing.
How to use it with your peers then? What if a users gets deleted..can I migrate the documents? Nothing on that.

But it is sharing everyones IP with Google for using fonts that could easily be part of the installation. :-(
I can not send anyone a link to sign a document because I would require the persons OK to share data with Google.

Are there any intentions of making OpenSign useable for everyone or is it just a sales tool so we get yet another subscription for a service?

**Comment by nxglabs** [comment]
Currently, we don't have a user management feature, but it will be available soon. Thank you for bringing the Google Fonts issue to our attention; we will fix it soon.

Please be patient with us.

**Reply by jmbxw2** [reply]
If you do not have user management, can't you clearly state what options are available for creating users? Are you guys being secretive on purpose? It seems like you're having second thoughts about the whole open source thing.

**Reply by andrew-opensignlabs** [reply]
Are you being disrespectful on purpose?Copy pasting comments on multiple discussions doesnt help. It will be helpful if you contribute to the project rather than wasting efforts writing such comments. Before v2.0.0 the only way to create new users was to sign up, now we have user management feature available. It has already been discussed on discord, documentation is also available & release notes already have instructions to upgrade. I have already replied to your comment on the other thread.
